microshift_networking/microshift-cni.adoc

```diff
@@ -6,7 +6,22 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-OVN-Kubernetes is the default networking solution for {microshift-short} deployments. OVN-Kubernetes is a virtualized network for pods and services that is based on Open Virtual Network (OVN). The OVN-Kubernetes Container Network Interface (CNI) plugin is the network plugin for the cluster. A cluster that uses the OVN-Kubernetes network plugin also runs Open vSwitch (OVS) on the node. OVN configures OVS on the node to implement the declared network configuration.
+The OVN-Kubernetes Container Network Interface (CNI) plugin is the default networking solution for {microshift-short} clusters. OVN-Kubernetes is a virtualized network for pods and services that is based on Open Virtual Network (OVN).
+
+* Default network configuration and connections are applied automatically in {microshift-short} with the `microshift-networking` RPM during installation.
+* A cluster that uses the OVN-Kubernetes network plugin also runs Open vSwitch (OVS) on the node.
+* OVN-K configures OVS on the node to implement the declared network configuration.
+* Host physical interfaces are not bound by default to the OVN-K gateway bridge, `br-ex`. You can use standard tools on the host for managing the default gateway, such as the Network Manager CLI (`nmcli`).
+* Changing the CNI is not supported on {microshift-short}.
+
+Using configuration files or custom scripts, you can configure the following networking settings:
+
+* You can use CIDR to route data packets to your devices. Routers retrieve the network and host address as specified by the CIDR suffix.
+* You can change the maximum transmission unit (MTU) value.
+* You can configure firewall ingress and egress.
+* You can configure network policies, including ingress and egress rules.
```

Comment:
The network policies shall be defined in the Microshift cluster, instead of on the host.